.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,43 @@
+## Description
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- The PR title should summarize the changes, for example "Add new group argument for the
+  pair plot".  Avoid non-descriptive titles such as "Addresses issue #348". If your pull 
+  request addresses an issue, please use the pull request title to describe 
+  the issue and mention the issue number in the pull request description.
+
+- The description should provide at least 1-2 sentences describing the pull request
+  in detail (Why is this change required?  What problem does it solve?) and
+  link to any relevant issues. If modifying a plot, render your plot to inspect for changes 
+  and copy image in the pull request message on Github
+
+- Please prefix the title of incomplete contributions with [WIP] (to indicate a work in 
+  progress). WIPs may be useful to (1) indicate you are working on something to avoid 
+  duplicated work, (2) request broad review of functionality or API, or (3) seek collaborators.
+-->
+<!-- Briefly describe the bug you are trying to fix -->
+
+<!-- Describe your changes in detail -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Is this PR following the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format (described below)?
+- [ ] Has added unit tests to prevent issue recurrence( using [pytest fixture pattern](
+      (https://docs.pytest.org/en/latest/fixture.html#fixture))?
+- [ ] Is code style correct (are you using pylint, black)?
+
+<!--
+Also, read the contributing guidelines carefully before submitting the PR. It is available 
+at https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md.
```

Comment:
Also, please consider reading the contributing guidelines and code of conduct carefully before... I liked having the code of conduct here like in the previous commit :)